core/cibox-project-builder/files/drupal7/scripts/updates.yml

```diff
@@ -0,0 +1,26 @@
+---
+# This playbook should be executed using ansible-playbook command
+# Developed for ansible version >= 1.7
+- hosts: localhost
+  connection: local
+  gather_facts: no
+
+  vars:
+    # this variable will be overridden from jenkins.
+    workspace_root: .
+    artifacts_file: commentinfo.md
+    build_reports_dir: build_reports
+    base_url: http://drupal.192.168.56.132.xip.io # WITHOUT TRAILING SLASH
```

Comment:
Do we have `http://drupal.192.168.56.132.xip.io` anywhere else? I'm not really happy with such duplicating.